test/behavior/tendrl/cluster_unmanaged.feature

```diff
@@ -0,0 +1,129 @@
+Feature: CLUSTER_UNMANAGED
+
+  Scenario: User sees cluster lists page after logging in
+    Given User opens the Tendrl link
+     When User passes correct login information
+     Then User should see "cluster list" page
+
+  Scenario: User sees cluster is unmanaged
+    Given User is on the "cluster list" page
+      And has at least one cluster available
+     Then User should see "No" under Managed
+
+  Scenario: User sees cluster status icon
+    Given User is on the "cluster list" page
+      And has at least one cluster available
+     When User hovers over cluster status icon
+      And User should see status is unknown
+
+  Scenario: User sees the cluster name/uuid
+    Given User is on the "cluster list" page
+      And has at least one cluster available
+     Then User should see UUID of unimported cluster
+
+  Scenario: User sees Cluster Version info
+    Given User is on the "cluster list" page
+      And has at least one cluster available
+     Then User should see Cluster Version info
+
+  Scenario: User sees hosts information
+    Given User is on the "cluster list" page
+      And has at least one cluster available
+     When User clicks on hosts link
+     Then User should be brought to hosts page
+
+  Scenario: Cluster is unavailable for import
+    Given User is on the "cluster list" page
+      And has at least one cluster available
+     When the cluster is unavailable for import
+     Then the import button should be unusable
+
+  Scenario: Cluster is available to import
+    Given User is on the "cluster list" page
+      And has at least one cluster available
+     When the cluster is available for import
+     Then the import button should be usable
```

Comment:
`     Then the import button should be available in kebab menu`

Reply:
Should we still classify the import button as being associated with the kebab menu even though it is a standalone button?
![before_import](https://user-images.githubusercontent.com/11084059/46550442-1279d980-c8a3-11e8-915a-8962f5d383ed.png)